virtcontainers/qemu.go

```diff
@@ -559,6 +559,28 @@ func (q *qemu) startSandbox() error {
 	return nil
 }
 
+func (q *qemu) waitAgent(eventCh chan govmmQemu.QMPEvent, timeout time.Duration) error {
+	for {
+		select {
+		case <-time.After(timeout):
+			return fmt.Errorf("Failed to wait agent (timeout %fs)", timeout.Seconds())
+		case ev := <-eventCh:
+			q.Logger().Infof("Receive event %s", ev)
+			if ev.Name == "VSERPORT_CHANGE" {
```

Comment:
Have you considered adding this constant to govmm and calling it from there?

Reply:
@BetaXOi great! I like this, but I agree with @ydjainopensource this event should be a constant

Reply:
Thank you, i will change this in next commit.